projects/storefrontapp-e2e-cypress/cypress/helpers/product-configurator-cart.ts

```diff
@@ -9,6 +9,16 @@ import * as configurationVc from './product-configurator-vc';
 
 const cartItemQuantityStepperSelector = '.cx-value cx-item-counter';
 
+/**
+ * Alias used for patching the quantity
+ */
+export const PATCH_QUANTITY_ALIAS = '@patchQuantity';
```

Comment:
Please rename so that it gets clear that this about changing quantity only in the cart context